fastlane/metadata/i18ncli.js

```diff
@@ -0,0 +1,320 @@
+  "ro",
+  "ru",
+  "sk",
+  "sv",
+  "th",
+  "tr",
+  "uk",
+  "vi",
+  "zh-Hans",
+  "zh-Hant",
+  "appleTV",
+  "iMessage",
+  "default"
+];
+
+const IOS_MAPPINGS = {
+  ar: "ar-SA",
+  de: "de-DE",
+  fr: "fr-FR",
+  en_AU: "en-AU",
+  en_CA: "en-CA",
+  en_GB: "en-GB",
+  en: "en-US",
+  es: "es-ES",
+  es_MX: "es-MX",
+  fr_CA: "fr-CA",
+  nl: "nl-NL",
+  pt: "pt-PT",
+  pt_BR: "pt-BR"
+};
+
+const SUPPORTED_ANDROID_METADATA_LOCALES = [
+  "af",
+  "sq",
+  "am",
+  "ar",
+  "hy-AM",
+  "az-AZ",
+  "bn-BD",
+  "eu-ES",
+  "be",
+  "bg",
+  "my-MM",
+  "ca",
+  "zh-HK",
+  "zh-CN",
+  "zh-TW",
+  "hr",
+  "cs-CZ",
+  "da-DK",
+  "nl-NL",
+  "en-AU",
+  "en-CA",
+  "en-GB",
+  "en-IN",
+  "en-SG",
+  "en-US",
+  "en-ZA",
+  "et",
+  "fil",
+  "fi-FI",
+  "fr-CA",
+  "fr-FR",
+  "gl-ES",
+  "ka-GE",
+  "de-DE",
+  "el-GR",
+  "gu",
+  "iw-IL",
+  "hi-IN",
+  "hu-HU",
+  "is-IS",
+  "id",
+  "it-IT",
+  "ja-JP",
+  "kn-IN",
+  "kk",
+  "km-KH",
+  "ko-KR",
+  "ky-KG",
+  "lo-LA",
+  "lv",
+  "lt",
+  "mk-MK",
+  "ms-MY",
+  "ms",
+  "ml-IN",
+  "mr-IN",
+  "mn-MN",
+  "ne-NP",
+  "no-NO",
+  "fa",
+  "fa-AE",
+  "fa-AF",
+  "fa-IR",
+  "pl-PL",
+  "pt-BR",
+  "pt-PT",
+  "pa",
+  "ro",
+  "rm",
+  "ru-RU",
+  "sr",
+  "si-LK",
+  "sk",
+  "sl",
+  "es-419",
+  "es-ES",
+  "es-US",
+  "sw",
+  "sv-SE",
+  "ta-IN",
+  "te-IN",
+  "th",
+  "tr-TR",
+  "uk",
+  "ur",
+  "vi",
+  "zu"
+];
+
+const ANDROID_MAPPINGS = {
+  "af-ZA": "af",
+  "sq-AL": "sq",
+  "ar-SA": "ar",
+  "be-BY": "be",
+  "bg-BG": "bg",
+  "ca-ES": "ca",
+  "he-IL": "iw-IL",
+  "hr-HR": "hr",
+  "et-EE": "et",
+  "fil-PH": "fil",
+  "gu-IN": "gu",
+  "id-ID": "id",
+  "kk-KZ": "kk",
+  "lv-LV": "lv",
+  "lt-LT": "lt",
+  // TODO: change in crowdin?
+  "nb-NO": "no-NO",
+  "pa-IN": "pa",
+  "ro-RO": "ro",
+  "rm-CH": "rm",
+  "sr-CS": "sr",
+  "sk-SK": "sk",
+  "sl-SI": "sl",
+  "sw-KE": "sw",
+  "th-TH": "th",
+  "uk-UA": "uk",
+  "ur-PK": "ur",
+  "vi-VN": "vi",
+  "zu-ZA": "zu"
+};
+
+function mapLanguageCodeToSupportedDirectoryName( languageCode ) {
+  return IOS_MAPPINGS[languageCode];
+}
+
+async function renameDirectories( ) {
+  // Get all directories in fastlane/metadata/ios
+  const iosDirectories = await fsp.readdir( path.join( __dirname, "ios" ), {
+    withFileTypes: true
+  } ).then( files => files.filter( file => file.isDirectory( ) ).map( file => file.name ) );
+  console.log( "Current list of the directories in fastlane/metadata/ios", iosDirectories );
+  await Promise.all( iosDirectories.map( async directory => {
+    const locale = mapLanguageCodeToSupportedDirectoryName( directory );
+    if ( !locale ) return;
+    const directoryPath = path.join( __dirname, "ios", directory );
+    const newDirectoryPath = path.join( __dirname, "ios", locale );
+    // Remove a potentially existing directory with the same name
+    try {
+      await fsp.rmdir( newDirectoryPath, { recursive: true } );
+      console.log( "Removed existing directory", newDirectoryPath );
```

Comment:
Might be ok with a `verbose` flag, but not by default, please.

Reply:
Behind flag now